src/pages/guides/application_state.md

```diff
@@ -159,23 +159,24 @@ In case of exceeding the usage limits, the State service will return with 429s.
 
 **Storage**
 
-- 100K key-values pairs
-- 1GB storage usage: `2 * key_sizes + value_sizes`
+- 200K key-values pairs.
+- 1GB for production Workspaces.
+- 200MB for non-production Workspaces.
 
-In case of exceeding the storage limits the service will return with an error and you will have to delete keys or wait for expiration to resume writing.
+Storage usage is calculated as `2 * key_sizes + value_sizes`
```

Comment:
I'm not sure if everybody remembers the order of operations but it will be unambiguous if you put brackets here:
`(2 * key_sizes) + value_sizes`